DigitalOffice/Tests/CheckRightsServiceTests/CommandsTests/GetRightsListCommandTests.cs

```diff
@@ -0,0 +1,27 @@
+using CheckRightsService.Commands;
+using CheckRightsService.Commands.Interfaces;
+using CheckRightsService.Repositories.Interfaces;
+using Moq;
+using NUnit.Framework;
+
+namespace CheckRightsServiceUnitTests.CommandsTests
+{
+    public class GetRightsListCommandTests
+    {
+        private Mock<ICheckRightsRepository> repositoryMock;
+        private IGetRightsListCommand command;
+
+        [SetUp]
+        public void Setup()
+        {
+            repositoryMock = new Mock<ICheckRightsRepository>();
+            command = new GetRightsListCommand(repositoryMock.Object);
+        }
+
+        [Test]
+        public void GetRightsListCommandSuccess()
+        {
+            Assert.DoesNotThrow(() => command.Execute());
```

Comment:
Add setup to repositoryMock which return something.

Reply:
fixed